packages/cli/src/lib/SchemaComposer.ts

```diff
@@ -64,17 +64,13 @@ export class SchemaComposer {
       output,
     };
 
-    for (const name of Object.keys(schemaNamedPaths)) {
-      const schemaPath = schemaNamedPaths[name];
-      const schemaFile = getSchemaFile(schemaPath);
-
-      if (!schemaFile) {
-        throw Error(`Schema "${name}" cannot be loaded at path: ${schemaPath}`);
-      }
-
-      options.schemas[name] = schemaFile;
+    const schemaFile = getSchemaFile(schemaNamedPath);
+    if (!schemaFile) {
+      throw Error(`Schema cannot be loaded at path: ${schemaNamedPath}`);
     }
 
+    options.schemas["main"] = schemaFile;
```

Comment:
Note: This should change to just schema - But left it that way (for now) to make it compliant with `composeSchema` interface